peer/node/start.go

```diff
@@ -871,7 +873,7 @@ func secureDialOpts() []grpc.DialOption {
 // 2. Init the message crypto service;
 // 3. Init the security advisor;
 // 4. Init gossip related struct.
-func initGossipService(policyMgr policies.ChannelPolicyManagerGetter, metricsProvider metrics.Provider,
+func initGossipService(mspID string, policyMgr policies.ChannelPolicyManagerGetter, metricsProvider metrics.Provider,
```

Comment:
why do you need to put here this mspID in this `initGossipService` ? 

You have the identity passed (`serializedIdentity`) and below you also have the `secAdv` as well.

Reply:
ah oops I missed this one, I meant to remove all of the passed mspIDs in the gossip path after we discussed the `secAdv` but forgot about this one. Thanks for catching it!